src/components/prettier.ts

```diff
@@ -6,19 +6,28 @@ export const prettierIt = (code: string) => {
     const prefix = "type __PrettyTsErrorType__ =";
     res = prefix + res;
     const temps: string[] = [];
-    // For: ... code?: string | /* 1 more */ | undefined; ...
-    res = res.replace(/\|\u0020\/\*\u0020\d\u0020more\u0020\*\//, (match) => {
-      temps.push(match);
-      return "/**\0*/";
-    });
+    // For:
+    // 1. ... code?: string | /* 1 more */ | undefined; ...
+    // 2. ... ; /* 1 more */; ...
+    //
+    res = res.replace(
+      /((\|)|(;))\u0020\/\*\u0020\d+\u0020more\u0020\*\/(?=((\|)|(;)))/g,
+      (match) => {
+        temps.push(match);
+        return "/**\0*/";
+      }
+    );
```

Comment:
The original `pretty-ts-errors` package seemed to have already handled converting `... xx more ...` so I wonder if there is a simpler way to fix it instead of adding a bunch of extra code...

Reply:
@rchl OK, I've probably figured out where my version is lacking, I've completely ignored the second argument to `formatDiagnostic`, `format`, which allows us to prettier the ts code and not care about `more`, thanks for pointing this out, I'll remove this part of my custom prettier code later on!